cmsl1t/config.py

```diff
@@ -227,10 +227,12 @@ def __fill_outdir_and_reload_files(self, reload_histograms, hist_files):
                     # Essentially, this is a new analysis output
                     output_folder = get_unique_out_dir(output_folder)
 
+        plots_folder = output_folder + "_plots"
+        plots_folder = os.path.realpath(plots_folder)
         output_folder = os.path.realpath(output_folder)
-        plots_folder = os.path.join(output_folder, "plots")
+        #plots_folder = os.path.join(output_folder, "plots")
         cfg['output']['folder'] = output_folder
-        cfg['output']['plots_folder'] = get_unique_out_dir(plots_folder)
+        cfg['output']['plots_folder'] = plots_folder
```

Comment:
I'm not a fan of this.  I think it's better that we put plots in a sub-dir of the same output directory.  I also think the unique plot directory part is good, although I think it doesn't work properly at this point, so I think it's ok for you to remove that bit.

Reply:
Yeah, I think it's neater that way too -- the issue is if one wants to easily cp/scp the output without the (potentially large) .root files it would be good (or at least easy) to have the plots directories separate in that case...

Reply:
I don't think that's a good reason either.  With a proper set of wildcards, you can get around this issue.  And besides if you just want the plots, then just copy the plot directory, not the whole output directory?  Anyway for `cp` you could 
```
shopt -s extglob
cp output_directory/**png destination_dir
```
or 
```
cp $(find output_dir -name '*png') destination_dir
```
or
```
shopt -s extglob
cp output_dir/plot_dir/!(*root) destination_dir
```
With scp or rsync:
```
scp 'lxplus:output_dir/plot_dir/*{png,pdf}' destination_di
```
or any of the commands above, if you escape things properly and make sure ext-glob is enabled in your bashrc file on the remote side.

Point being: your problem doesn't need this change, and the files and book-keeping is much better without it.

Reply:
I agree tbh, I was mainly pushing this so another user could try it but forgot it would update the pull request too!

I agree that having plots_v1 etc *inside* the output_dir is much less confusing and much better for book-keeping etc. I can understand the desire for a simple way to copy lots of jobs' worth of plots (including the folder hierarchy etc) but not the .root files -- However the commands you've listed get around that nicely and it's not worth messing up the folder structure for.

In fact, I've just tried rsync (where have I been without it?!) and did this:
```
rsync -avR <user>@lxplus.cern.ch:/path/to/cms-l1t-analysis/output/./<dirs>/plot* <destination>
```
which worked beautifully and just gave me everything after "/./" with the plots subdirectories and no .root files :-)

So let's not keep this change to the code but I'll put a couple of those example commands in the instructions/docs to keep the people happy ;-)